src/widget_type.h

```diff
@@ -105,6 +108,14 @@ enum WidgetType {
 	NWID_PUSHBUTTON_DROPDOWN = NWID_BUTTON_DROPDOWN | WWB_PUSHBUTTON,
 };
 
+/**
+ * Base widget flags.
+ */
+enum WidgetBaseFlags : uint8 {
```

Comment:
You have to wonder whether an enum with a single value is worth the bother. Why not just a boolean flag? It can always be converted to an enum later

Reply:
I expect @JGRennison had plans to add another flag in his patch pack. At least, I assume that would be the reason?